### DISCUSSION TITLE
Issue Tracker with AIR- prefix

### DISCUSSION THREAD

**Discussion by aW4KeNiNG** [airsdk/Adobe-Runtime-Support#966] [Q&A] [answered]
Hi.

Where can I check more information about the AIR-NUMBER issue in the release notes?

Thanks in advance.

Regards.

**Comment by ajwfrost** [comment, marked as answer]
That’s a reference into our internal/ private system: what number was it you wanted information on?

**Reply by aW4KeNiNG** [reply]
AIR-207: Additional code-signing options can be passed in to ADT When creating a macOS ‘bundle’
package. This would be the ‘native’ signing options, after the ‘-target bundle’ parameters, and would need to
be a Keychain store type with the alias being the name of the certificate to use in code-signing. Using this
means that the output bundle should be fully structured and code-signed, and ready for notarization.

What is the parameter name? Any example?

**Reply by aW4KeNiNG** [reply]
@ajwfrost any info?

**Reply by ajwfrost** [reply]
Sorry - missed responding to this.

So, currently if you build a 'bundle' target on macos:
```
$ adt -package  -storetype pkcs12 -keystore ../harman.p12 -target bundle MyOutput.app application.xml HelloTriangle.swf
```
then the resulting app bundle is not signed properly:
```
% codesign -v -vvv MyOutput.app
MyOutput.app: invalid Info.plist (plist or signature have been modified)
In architecture: arm64
```

Rather than you having to then work out how to sign everything, you can provide an additional signing key from the macOS keychain as a 'native' signing parameter:
```
$ adt -package -storetype pkcs12 -keystore ../harman.p12 -target bundle -storetype KeychainStore -alias "Developer ID Application: Harman" SignedApp.app application.xml HelloTriangle.swf
```
and this will then use the native `codesign` tool to find and sign the appropriate binaries within the bundle:
```
% codesign -v -vvv SignedApp.app
--prepared:/Users/ukinanfr/Adobe/Testing/HelloTriangle/SignedApp.app/Contents/Frameworks/Adobe AIR.framework/Versions/Current/.
--validated:/Users/ukinanfr/Adobe/Testing/HelloTriangle/SignedApp.app/Contents/Frameworks/Adobe AIR.framework/Versions/Current/.
SignedApp.app: valid on disk
SignedApp.app: satisfies its Designated Requirement
```

The idea being: you can then just do the notarization on this .app bundle and it should just work first time...

thanks

**Reply by aW4KeNiNG** [reply]
thanks!!
